cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -115,6 +114,10 @@
  * <p>
  * Most of these operations are no-ops as many of these methods and pretty closely coupled to the Hadoop implementation
  * and have no analogy in Kubernetes.
+ * <p>
+ * If {@link TwillSpecification} contains multiple {@link TwillRunnable}, the first runnable will be treated as the
+ * main container, and the rest will be treated as sidecar containers.
+ * TODO This assumption needs to be changed by using {@ TwillSpecification.PlacementPolicy}.
```

Comment:
If this TODO wont be addressed in the next couple PRs then we should create a JIRA and put the ID here:
 `TODO (CDAP-XXX): <description>`

Reply:
Sure thing. Will create a JIRA ticket and put its ID here once the approach gets approved.